tools/respecDocWriter.js

```diff
@@ -235,9 +244,13 @@ function makeConsoleMsgHandler(page) {
         // https://github.com/GoogleChrome/puppeteer/issues/1939
         return;
       }
-      const abortOnWarning = whenToHalt.haltOnWarn && type === "warning";
-      const abortOnError = whenToHalt.haltOnError && type === "error";
       const output = `ReSpec ${type}: ${colors.debug(text)}`;
+      if (type === "error") {
```

Comment:
Nit: use switch statement here I think.

Reply:
Maybe can be merged with the following switch.

Reply:
Thanks for the advice! Already fixed.